src/store/inbox/inboxSlice.ts

```diff
@@ -0,0 +1,52 @@
+import { createSlice, createEntityAdapter } from '@reduxjs/toolkit';
+import type { Inbox } from '@/types/Inbox';
+import { inboxActions } from './inboxActions';
+
+const DEFAULT_INBOX: Readonly<Inbox> = {
+  id: 0,
+  name: 'All Inboxes',
+  channelType: 'Channel::All',
+  avatarUrl: '',
+  channelId: 0,
+  phoneNumber: '',
+  medium: 'Channel::All',
```

Comment:
This doesn't seem like something that should be in the state. I believe this is used to render the option All in the filter. It is a UI logic, shouldn't be part of the global state.